logger.go

```diff
@@ -0,0 +1,9 @@
+// +build gosnmp_nodebug
+
```

Comment:
Please also add the instructions on how to disable the logger via a build tag here

Reply:
Fixed